src/test/java/com/regnosys/rosetta/common/serialisation/json/projection/JsonProjectionDataLoaderTest.java

```diff
@@ -1,9 +1,10 @@
 package com.regnosys.rosetta.common.serialisation.json.projection;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
 import com.regnosys.rosetta.common.serialisation.RosettaObjectMapper;
 import com.regnosys.rosetta.common.serialisation.projectiondata.JsonProjectionDataLoader;
-import com.regnosys.rosetta.common.serialisation.projectiondata.ProjectionDataSet;
+import com.regnosys.rosetta.common.serialisation.DataSet;
 import com.regnosys.rosetta.common.serialisation.reportdata.ReportDataItem;
 import org.junit.jupiter.api.Test;
 
```

Comment:
Do we need this test now? What is the difference between this one and the one in reports?